frontend/components/Domain/Recipe/RecipeTimeline.vue

```diff
@@ -31,7 +74,7 @@
     </div>
     <v-card v-else-if="!loading">
```

Comment:
I would add some top margin here so top of the card does not lie on top of the divider. 

```suggestion
    <v-card v-else-if="!loading" class="mt-2">
```